website/docs/r/application_gateway.html.markdown

```diff
@@ -379,6 +381,28 @@ A `url_path_map` block supports the following:
 
 * `path_rule` - (Required) One or more `path_rule` blocks as defined above.
 
+---
+
+A `ssl_policy` block supports the following:
+
+* `policy_type` - (Optional) The Type of the Policy. Required if `policy_name` is set. Possible values are `Predefined` and `Custom`. Not compatible with `disabled_protocols`.
+
+For `policy_type`=`Predefined`:
+
+* `policy_name` - (Optional) The Name of the Policy e.g AppGwSslPolicy20170401S. Required if `policy_type` is set to `Predefined`. Possible values can chang over time and 
+are published here https://docs.microsoft.com/de-de/azure/application-gateway/application-gateway-ssl-policy-overview. Not compatible with `disabled_protocols`.
```

Comment:
The English version if that helps https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-ssl-policy-overview

Reply:
@jstewart612 thanks. How could that happen :P.

Reply:
It is a genuine mystery to one and all!  ;)

In all seriousness, I don't think this provider's documentation is localized to any other languages, and thus, I think most are, perhaps unfairly, expecting off-world links to go to sites in the same language.

It's the most minor of minor points and really anyone who knows enough how to use this ought know how to go find the English version.... but yeah.

Reply:
@jstewart612 anyways fixed it :) 👍

Reply:
the Azure docs auto redirect to the users locale if you leave off the locale prefix, so we can make this: `https://docs.microsoft.com/azure/application-gateway/application-gateway-ssl-policy-overview` :)